threejs/fxhash_Rooms/src/figures.js

```diff
@@ -209,9 +208,65 @@ const genererFigures = (fxhash) => {
       hatch: true, // Fill with white texture
       full: false, // Fill with color texture (in the anaverse, red and cyan)
     });
+
+    // "create" the vertical column members to extend through the ground plane
+    figures.push({
+      geometry: { type: "BoxGeometry", args: [0.1, 20, 0.1] },
+      pos: {
+        // Anaglypic Question
+        // Can we add a column that runs from each position of the original room polyline down through the ground plane? 
+        // Position for each beam should be the position of each splinePt used to create the polygon from the floor generator
+        x: px * u, 
+        y: py * u - 20/2, // <---- Arbitrary column depth should match the column height and should pass through the ground plane
+        z: pz * u,
+      },
+      rot: {
+        // Rotation
+        x: 0,
+        y: Math.sin(fxrand() * Math.PI * 2),
+        z: 0,
+      },
+      scale: {
+        x: 1,
+        y: 1,
+        z: 1,
+      },
+      name: "column",
+      lines: true, // Display color segments (like wireframe, but faces not triangles)
+      hatch: true, // Fill with white texture
+      full: false, // Fill with color texture (in the anaverse, red and cyan)
+    });
   }
 
-  // add the rooms
+  // Anaglyphic Question - Can we add a single ground plane? I can't seem to get this working
```

Comment:
I had a bit of trouble adding more geometry to the figures.js file.

I'd like to add a landing pad where the user first spawns in the anaverse. As well as a ground plane that acts as a clip for the extended columns. I'm also open to how you accomplished this in the WED project if that feels a bit more spatial interesting.

![image](https://user-images.githubusercontent.com/54593392/185296225-4101868f-f708-422a-ace2-b95d2171388b.png)

Reply:
that being, no groud plane and all columns extending to the same depth.